stock_picking_zone/models/stock_picking_type.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2019 Camptocamp (https://www.camptocamp.com)
+
+from odoo import _, api, exceptions, fields, models
+
+
+class StockPickingType(models.Model):
+    _inherit = 'stock.picking.type'
+
+    is_zone = fields.Boolean(
```

Comment:
We'll probably want to rename this (as well as the module's name) by something more specific than `zone`.